opencti-platform/opencti-front/src/utils/Entity.ts

```diff
@@ -1,10 +1,9 @@
 export const resolveLink = (type: string): string | null => {
-  switch (type) {
+  const upperCasedType = `${type[0].toUpperCase()}${type.slice(1)}`;
+  switch (upperCasedType) {
```

Comment:
This won't cover all typos, but this one is quite common.

Reply:
I'm not sure to understand why we need to uppercase the first letter, for which cases do we need to ?

Reply:
I've removed this tweak, this is irrelevant.

We shall use the right entity_type and that's it.
It's more a Typescript issue at this point (not setting prop entityType to `string` but the enum from GQL.